test_t3 of Test_Rrecord_with_resources_used is failing while verifying accounting logs

Mahalty opened the pull request:
<!--- Please review your changes in preview mode -->
<!--- Provide a general summary of your changes in the Title above -->

#### Bug
* Test "test_t3" of "Test_Rrecord_with_resources_used" is failing while verifying the log-msg in accounting logs
* Change name of the tests according to PTL naming convention

#### Affected Platform(s)
* ALL

#### Cause / Analysis / Design
* The intent of the test is to verify that R record appears only for rerunnable jobs.
* In the test we are submitting 3 jobs(J1: rerunnable jobs with -koe option, J2: Non rerunnable job with -koe option and J3: rerunnable job array) and the test is expecting that a R record entry should exist for J2 job which is not true. As it is non-rerunnable job R record will not appear in the accounting logs

#### Solution Description
* Pass parameter "existence=False" in accounting_match function for J2.

#### Testing logs/output
* [Complete_Execution_logs_Rrecord.txt](https://github.com/PBSPro/pbspro/files/2817526/Complete_Execution_logs_Rrecord.txt)


#### Checklist:
<!--- Use the preview button to see the checkboxes/links properly. -->
<!--- Go over the following points, and put an `x` (without spaces around it) in the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to this pull request** as evidence of testing/verification.


__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

anamikau commented on test/tests/functional/pbs_Rrecord_resources_used.py:
```diff
@@ -404,7 +404,7 @@ def test_t5(self):
             id=jid3s1, regexp=True)
 
-    def test_t6(self):
+    def test_Rrecord_with_select_and_multiple_rerun_of_job(self):
```
please make this name little short "test_Rrecord_with_multiple_reruns_select" or "test_Rrecord_with_multiple_reruns_case2"

anamikau commented on test/tests/functional/pbs_Rrecord_resources_used.py:
```diff
@@ -303,7 +303,7 @@ def test_t4(self):
                             expect=True)
 
-    def test_t5(self):
+    def test_Rrecord_with_multiple_rerun_of_job(self):
```
update to "test_Rrecord_with_multiple_reruns"

anamikau commented on test/tests/functional/pbs_Rrecord_resources_used.py:
```diff
@@ -194,7 +194,7 @@ def common(self, is_nonrerunnable, restart_mom):
         return jid1, jid2, jid3s1
 
-    def test_t1(self):
+    def test_Rrecord_when_mom_down_and_nodefailrequeue(self):
```
I suggest just change it to "test_Rrecord_with_nodefailrequeue"

Mahalty commented:
Thanks @prakashcv13 @anamikau and @bhroam  for the review.
I have addressed review comments. Please review

**Latest execution-logs:**
[Latest_Execution_logs_Test_Rrecord_with_resources_used.txt](https://github.com/PBSPro/pbspro/files/2820468/Latest_Execution_logs_Test_Rrecord_with_resources_used.txt)

Mahalty commented:
Thanks @bhroam @anamikau @agrawalravi90  for the review. 
@PBSPro/pbspro-maintainers  I have squashed my commits and rebased my branch. Please take this PR forward
